### DISCUSSION TITLE
What Versions / Environments are you using? - R, R-studio, Python, OS, etc

### DISCUSSION THREAD

**Discussion by ergonyc** [ndcn/omicser#174] [General Survey]
Hey everyone, I thought it would be helpful to know where everyone is at.

What version:
- R
- R-studio
- python

Operating System:
- Linux
- Windows
- Mac
- Docker?

Etc: (what else are you using?)

**Comment by ergonyc** [comment]
Here's what I'm ( @ergonyc  ) using:

Versions:
- R: 4.1.0
- R-studio:  1.4.1717
- python:  3.8, 3.9

Operating System:
- Linux: Ubuntu 20.10 (pop!_os) and 20.04
- Mac: Monterey, Catalina
- Docker: occationally for testing

Etc: (what else are you using?)
- VSCODE

**Comment by ricoderks** [comment]
I'am using:

What version:

- R v4.1.2
- R-studio v2021.09.0 Build 351
- python v3.9.7

Operating System:

- Linux: Ubuntu 20.04 LTS

I have the latest version running in a separate docker container for everybody to access. 

Docker:

- Docker  v20.10.12, build e91ed57
- R v4.1.2
- python v3.8.12

Cheers, Rico
